### PR TITLE
Models tests

### PR DESCRIPTION
**Pull request by AlexandrZagorskiy**

### CODE REVIEW

File: tests/models_test.py [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+            t.EDGE_ACCESS_VAR_POS_TEMP,
+            t.EDGE_ACCESS_VAR_NEG_TEMP,
+            t.EDGE_ACCESS_VAR_FUZ_TEMP,
+        }
+        for sc_type in temp_types:
+            assert sc_type.is_temp()
+
+        for sc_type in self.types.difference(temp_types):
+            if sc_type.is_temp():
+                print(sc_type)
```
Comment (NikitaZotov) [inline tests/models_test.py]: What is it

File: tests/models_test.py [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+
+    def test_change_const(self):
+        assert t.CONST.change_const(True).is_const()
+        assert t.CONST.change_const(False).is_const() is False
+        assert t.CONST.change_const(True).is_var() is False
+        assert t.CONST.change_const(False).is_var()
+
+    def test_merge(self):
+        assert t.NODE_CONST.merge(t.NODE_CLASS) == t.NODE_CONST_CLASS
+        with pytest.raises(InvalidValueError, match=CommonErrorMessages.INVALID_VALUE.value):
```
Comment (NikitaZotov) [inline tests/models_test.py]: May be type?

**Review by NikitaZotov — COMMENTED** [review]
What is

File: tests/models_test.py (hunk 1 above)
Comment (AlexandrZagorskiy) [reply tests/models_test.py]: Nothing

**Review by NikitaZotov — APPROVED** [review]

**Pull request merged**